study-builder/studies/testboston/study-workflows.conf

```diff
@@ -35,7 +35,13 @@
         {
           "type": "ACTIVITY",
           "activityCode": ${id.act.address},
-          "expression": """user.studies["testboston"].forms["COVID_SURVEY"].isStatus("COMPLETE")"""
+          "expression": """
+            user.studies["testboston"].forms["COVID_SURVEY"].isStatus("COMPLETE")
+            && !user.studies["testboston"].forms["ADDRESS"].isStatus("COMPLETE")"""
+        },
+        {
+          "type": "DASHBOARD",
```

Comment:
After `covid_survey`, if the `address` activity is already done, then go back to the `dashboard`.